__tests__/OrderedSet.ts

```diff
@@ -59,4 +59,80 @@ describe('OrderedSet', () => {
       'CD',
     ]);
   });
+
+  it('ensure that `subtract` works correcly (see https://github.com/immutable-js-oss/immutable-js/issues/139 )', () => {
```

Comment:
```suggestion
  it('handles subtract when Set contains >=32 elements (https://github.com/immutable-js-oss/immutable-js/issues/139 )', () => {
```

Reply:
now that i’ve reviewed more of this PR, i’m thinking maybe this test case should be moved to `__tests__/issues.ts`